src/pencilkit.cs

```diff
@@ -2,9 +2,9 @@
 // PencilKit C# bindings
 //
 // Authors:
-//	TJ Lambert  <t-anlamb@microsoft.com>
+//	Whitney Schmidt  <whschm@microsoft.com>
```

Comment:
I would not remove TJs copyright. It is his original work, we should just add our names, example: https://github.com/xamarin/xamarin-macios/blob/main/src/Foundation/NSUrlSessionHandler.cs#L6

Reply:
Yep, do not remove copyright/authors from header files